content/release-2024-0/devices-2024-0.md

```diff
@@ -109,20 +109,44 @@ Build version</th>
 
 <tr>
 <td>Device Management</td>
+<td>Issues with the Ericsson DCP SMS provider when attempting to send an SMS have been resolved and outgoing requests are sent as expected to the Ericsson DCP API.</td>
+<td>DM-2215</td>
+<td>10.18.101.0</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>Device management</td>
+<td>Issues with flashing the device availability connection status have been fixed.</td>
+<td>MTM-51541</td>
+<td>10.18.106.0</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>Device management</td>
 <td>In the SmartREST template editor, issues with the presentation of the <b>External ID type</b> field for Inventory POST messages have been fixed. Under <b>CSV preview</b> the generated "Template creation CSV" has been adjusted to include the "ID", "externalId" and "externalIdType" values.</td>
 <td>DM-2093</td>
 <td>10.18.22.0</td>
 <td>UI</td>
 </tr>
 
 <tr>
-<td>Device Management</td>
+<td>Device management</td>
 <td>SmartREST Inventory GET templates created in the UI did not generate responses when there was no external ID type declared in the template. This issue has been addressed for both existing and newly created templates.</td>
 <td>DM-2126</td>
 <td>10.18.7.0</td>
 <td>Core</td>
 </tr>
 
+<tr>
+<td>LWM2M</td>
+<td>The configuration flag <code>fwResetStateMachineOnStart</code> has been added to control if the LWM2M agent resets the firmware update state machine on the client at the beginning of a firmware update. The default of this flag is true which matches the existing behaviour of the LWM2M agent. It is available in the [device registration settings](https://cumulocity.com/guides/protocol-integration/lwm2m/#device-registration-settings).</td>
```

Comment:
"true" is the value, we sometimes add monospace markup for that. not sure if we do that in release notes

Reply:
changed